api/nodes/views.py

```diff
@@ -40,7 +40,7 @@ def get_default_odm_query(self):
         return (
             Q('is_public', 'eq', True) &
             Q('is_deleted', 'ne', True) &
-            Q('is_dashboard', 'ne', True)
```

Comment:
Do we want to allow users to get folders through the `Node` search, or do we want to eventually treat folders like their own thing as we are now? If the former, which I don't agree with because we may eventually change folder implementation, then we should allow people to get their dashboards as well, because they'll need the dashboard folder for most of the same things they'll need the other folders for. If not, Dashboards are Folders, so this is the more correct case. Restricting only dashboards for anything other than DELETE or POST operations is incorrect.